frontend/src/modules/entitydetails/components/Metadata.js

```diff
@@ -149,7 +231,7 @@ export default class Metadata extends React.Component<Props> {
     render(): React.Node {
         const { entity, locale, openLightbox, pluralForm } = this.props;
 
-        return <div className="metadata">
+        return <div className="metadata" ref={ this.span }>
```

Comment:
Why do you reference this as a "span", if it's a div?

Reply:
I referenced this as a "span" as the original placement I had for it was on a span, and I didn't catch it and rename when I moved it to the div.  It is changed now.